.gitignore

```diff
@@ -35,6 +35,7 @@ pantheon.upstream.yml
 *.tar
 *.zip
 *.tgz
+!wp-includes/**/*.gz
```

Comment:
There is already a separate PR #175 for this - do we need the change duplicated here?

Reply:
@ataylorme That PR contains the same commits that I put in here. The new process for Circle testing is to merge into the `default` branch first -- the merge to master will happen automatically. We collect PRs together into a single PR to ship so that customers get just one dashboard notification on the upgrade rather than several.